src/Cooking.css

```diff
@@ -64,11 +64,9 @@
 }
 
 .Cooking-StepImage {
-  display: flex;
```

Comment:
if you delete display: flex, does align-items and justify-content still work? I would assume not, but I'd be happy to be wrong